pal/inc/pal_mstypes.h

```diff
@@ -200,7 +200,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use 
 // __int64 as though it were intrinsic
 
-#ifdef BIT64
+#ifndef BIT64
```

Comment:
@jianchun now it's clear what `BIT64` corresponds. In this case, isn't `_Is_integral` is defined twice for `signed and unsigned long` when BIT64 is set? See line 311 `pal/inc/type_traits`

BTW;PR is updated to just to take care of duplicate issue.

Is there any particular reason for not using `typedef int64_t __int64` instead of all these BIT64 stuff ?